WEEK1 SOLUTION/script4.py

```diff
@@ -0,0 +1,13 @@
+# write a script to check number is armstrong or not..!

```

Comment:
Here, in this program according to definition you first need to check whether the inputed number is integer number Or not..? So, where it is...?

Reply:
a = int(input("Enter any number: "))
Please check the above line.
According to the definition  i had used **int** datatype that take input of only **number** or **integer,**
And  i have passed specific **message** from that user will understand that they have to input only **numbers.** 
If the user put any other data instead of number then it will show an ERROR.